chrome.tera

```diff
@@ -0,0 +1,46 @@
+---
+whiskers:
+  version: 2.0.0
+  matrix:
+    - flavor
+    - accent
+  filename: "themes/{{ flavor.identifier }}/{{ accent }}/manifest.json"
+version: 4.1
+---
+{%- set palette = flavor.colors -%}
+{%- macro rgb(c) -%}
+[{{ c.rgb.r }}, {{ c.rgb.g }}, {{ c.rgb.b }}]
+{%- endmacro rgb -%}
+{
+  "manifest_version": 3,
+  "name": "Catppuccin Chrome Theme - {{ flavor.identifier | capitalize }} {{ accent | capitalize }}",
+  "description": "Soothing pastel theme for Google Chrome - Catppuccin {{ flavor.identifier | capitalize }}",
```

Comment:
```suggestion
  "name": "Catppuccin Chrome Theme - {{ flavor.name }} {{ accent | capitalize }}",
  "description": "Soothing pastel theme for Google Chrome - Catppuccin {{ flavor.name }}",
```
Same with the Google Store one if we are keeping that.

Reply:
I had kept it this way to avoid the accent on Frappe, don't want to run the risk of breaking things.